collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
+
+The amount of electricity produced is determined minute by minute by its consumption. Power stations are switched on and off according to demand to avoid voltage drops in the power grid. How the power plants are switched on and off depends mainly on how long it takes to start the plant. Nuclear plants take weeks to start up, coal plants can be up to full power in a matter of hours, and gas or hydroelectric plants, which can switch on and off in minutes, are used to smooth out short-term fluctuations in demand. Wind and solar power plants also contribute to the grid (when the wind is blowing and the sun is shining), and battery or other storage facilities can be expected to play an important role in the future. Another essential factor that determines which sources will produce electricity at any given time is the cost of running the source (for example, solar or wind plants produce virtually free electricity, while coal plants have to pay for coal) and the trading system (renewables have priority access to the grid and long-term guaranteed feed-in tariffs).
+
+**If reduced electricity consumption in households led to the shutdown of gas and coal-fired power plants, then for every kWh saved, about 1 kg of emissions would not be generated.** This value is based on Decree No. 480/2012 Coll. on Energy Audit and Energy Assessment and takes into account that a large part of household electricity is consumed at peak times, which often have to be covered by the most emitting sources of electricity. **A 50% reduction in consumption would therefore result in a saving of 7.6 million tonnes of <glossary id="co2eq">CO<sub>2</sub>eq</glossary>.** This is a rather exaggerated estimate of savings, as the real emission intensity for peak operation in 2020 is likely to be slightly lower than that set out in the abovementioned decree.
+
+Most household electricity is used for cooking and heating water (or heating if it is electric). Washing machines and refrigerators use less, and the least amount of electricity is used for lighting. However, electricity provides water heating in about 20% of Czech households and is used for cooking in more than 40% of households<!--(see [CSO statistics](https://www.czso.cz/documents/10180/50619982/ENERGO_2015.pdf), Table 1 - 3.3)-->. It is, therefore, questionable whether it is possible to achieve a significant reduction in household consumption. If inefficient boilers and stoves are used for heating, it is desirable to increase consumption to replace them with a heat pump, for example. Some savings could be achieved by preheating water with solar collectors or using surplus electricity from photovoltaic cells. Savings in cooking could be achieved, for example, by using an induction cooker. Replacing conventional light bulbs with LEDs or other more efficient light sources has little effect on electricity consumption.
```

Comment:
Unable to find this link in English. Not sure if there is an equivalent to https://www.czso.cz/documents/10180/50619982/ENERGO_2015.pdf